src/Games/NexusMods.Games.MountAndBlade2Bannerlord/NexusMods.Games.MountAndBlade2Bannerlord.csproj

```diff
@@ -26,4 +26,9 @@
     <InternalsVisibleTo Include="NexusMods.Games.MountAndBlade2Bannerlord.Tests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="MnemonicDB\SubModuleInfoExtended.cs" />
+    <Compile Remove="MnemonicDB\ModuleInfoExtendedWithPath.cs" />
+  </ItemGroup>
```

Comment:
This seems wrong.

Reply:
removed, Rider likes to add these sometimes instead of deleting files